src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -601,8 +601,6 @@ public abstract partial class TripleDES : System.Security.Cryptography.Symmetric
     {
         protected TripleDES() { }
         public override byte[] Key { get { throw null; } set { } }
-        public override System.Security.Cryptography.KeySizes[] LegalBlockSizes { get { throw null; } }
```

Comment:
In this case the base class method is just virtual, not abstract. I don't think this would cause a problem.

We aren't enforcing the real type to match the contract exactly w.r.t. overloads otherwise we would have previously had an issue with LegalBlockSizes.

Also if we are not allowed to remove overloads we would be forced to have many stub methods with a call to base._Foo_.